src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -0,0 +1,186 @@
+            cy.contains('Save').click();
+            cy.wait('@createModelingSubmission');
+            cy.get('#exam-navigation-bar').find('.btn-danger').click();
+            examStartEnd.finishExam();
+            cy.login(tutor, '/course-management/' + course.id + '/exams');
+            cy.contains('Assessment Dashboard', { timeout: 60000 }).click();
+            assessmentDashboard.startAssessing();
+            modelingAssessment.addNewFeedback(2, 'Noice');
+            modelingAssessment.openAssessmentForComponent(1);
+            modelingAssessment.assessComponent(1, 'Good');
+            modelingAssessment.openAssessmentForComponent(2);
+            modelingAssessment.assessComponent(0, 'Neutral');
+            modelingAssessment.openAssessmentForComponent(3);
+            modelingAssessment.assessComponent(-1, 'Wrong');
+            assessmentDashboard.submitAssessment();
+            cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+            cy.get('.question-options').contains('2 of 10 points').should('be.visible');
+        });
+
+        it('Assess a text exercise submission', () => {
+            const exerciseTitle = 'Cypress Text Exercise';
+            courseManagementRequests.createAndAddTextExerciseToExam(exerciseGroup, exerciseTitle);
+            courseManagementRequests.generateMissingIndividualExams(exam);
+            courseManagementRequests.prepareExerciseStartForExam(exam);
+            cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+            examStartEnd.startExam();
+            cy.contains(exerciseTitle).click();
+            cy.get('#text-editor-tab').type(textSubmission.text);
+            cy.contains('Save').click();
+            cy.get('#exam-navigation-bar').find('.btn-danger').click();
+            examStartEnd.finishExam();
+            cy.login(tutor, '/course-management/' + course.id + '/exams');
+            cy.contains('Assessment Dashboard', { timeout: 60000 }).click();
+            assessmentDashboard.startAssessing();
+            assessmentDashboard.addNewFeedback(7, 'Good job');
+            assessmentDashboard.submitAssessment();
+            cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+            cy.get('.question-options').contains('7 of 10 points').should('be.visible');
+        });
+    });
+
+    describe('Exam Programming Exercise Assessment', () => {
+        let programmingExerciseName: string;
+        let programmingExerciseShortName: string;
+        const examEnd = (Cypress.env('isBamboo') ? GROUP_SYNCHRONIZATION / 1000 : 0) + 115;
+
+        before('Generate exercise names', () => {
+            uid = generateUUID();
+            programmingExerciseName = 'Cypress programming exercise ' + uid;
+            programmingExerciseShortName = 'cypress' + uid;
+        });
+
+        beforeEach('Create Exam', () => {
+            prepareExam(dayjs().add(examEnd, 'seconds'));
```

Comment:
With the change from above
```suggestion
            prepareExam(dayjs().add(examEnd, 'milliseconds'));
```